docs/development/building-good-containers.md

```diff
@@ -0,0 +1,224 @@
+# Building Good Containers
+
+There are a number of recommendations based on our experience building
+Node.js applications in containers for deployments:
+
+* build non-root containers
+* avoid using trusted ports
```

Comment:
do you mean reserved/privileged ports here? i.e. 1-1023

Reply:
Yes, because of nodejs limitations with setcap (to enable non-root to use ports 1-1023).

Reply:
right--but i think the verbiage could be more precise (e.g. privileged vs. trusted)